webpack.config.babel.js

```diff
@@ -185,7 +185,8 @@ const baseConfig = {
     alias: {
       'react': 'preact-compat',
       'react-dom': 'preact-compat',
-      'react-modal': 'react-modal-onfido'
+      'react-modal': 'react-modal-onfido',
+      'utils': `${__dirname}/src/components/utils/`
```

Comment:
Pick whichever you prefer, but just fyi - I personally make aliases like this `~utils` instead
I find this is much more explicit, so when I'm reading
```
import React from "react";
import foo from "~bar";
```
I know that `bar` is just an alias to this repo's src, rather than a package in node_modules somewhere

Reply:
interesting. common standard?

Reply:
I've seen a few people do similar things. Not sure if it's "common". I've never understood why aliasing etc. isn't more common. `../../../../foo` is super common, and super annoying :(